src/attribution/onLCP.ts

```diff
@@ -33,13 +33,26 @@ const attributeLCP = (metric: LCPMetric) => {
     if (navigationEntry) {
       const activationStart = navigationEntry.activationStart || 0;
       const lcpEntry = metric.entries[metric.entries.length - 1];
+
+      // Should always have an lcpEntry to be able to attribute
+      if (!lcpEntry) return;
```

Comment:
This isn't needed as there's already an `if (metric.entries.length)` check above.